action.yml

```diff
@@ -2,15 +2,11 @@
 
 name: Set up MATLAB
 description: >-
-  Install MATLAB on a Linux-based GitHub-hosted agent. Currently, this action is
-  available only for public projects and does not include transformation
-  products, such as MATLAB Coder and MATLAB Compiler.
+  Setup MATLAB on a Linux-based GitHub-hosted agent
```

Comment:
Do we want to include Linux-based in the action description?

Reply:
I'm not sure 🤔 
Methinks if someone sees it up front, they'll be less likely to try it on a currently unsupported runner 🤔 
But I can also see the case for dropping it altogether too 🤔

Reply:
Oh, one more thing. GitHub-hosted "runner". Right?

Reply:
Thank you! Done 😄